Dockerfile

```
@@ -0,0 +1,8 @@
+FROM ruby:alpine
```

It would be nice to pin this image to a specific ruby version
```suggestion
FROM ruby:3.0.2-alpine
```